src/pages/projectCreate/hooks/useWalletForm.tsx

```diff
@@ -83,6 +85,8 @@ export const useWalletForm = ({
 
   const projectWallet = project?.wallets[0]
 
+  const [feePercentage, setFeePercentage] = useState<number>(projectWallet?.feePercentage || 0.6)
+
```

Comment:
If this is the initial default value for Node runners, let's get a variable for this value: 0.6.

We like to avoid use of Magic numbers, in the middle of the code. It's for better readability and maintainability.

Reply:
I just assumed that this would be the default variable as this was the one highlighted on the design, but it should be discussed with @MickMorucci to see what default fee percentage we want for node runners. 

Yes I agree, it's related to your previous comment. Having an enum for this will solve it.

Reply:
I think this was part of the requirement, let's put it 0 for now. And we'll confirm this with mick, we can change it again later.